manifests/client.pp

```diff
@@ -1,40 +1,42 @@
 # Setup for ossec client
 class wazuh::client(
-  $ossec_active_response      = true,
-  $ossec_rootcheck            = true,
-  $ossec_rootcheck_frequency  = 36000,
-  $ossec_rootcheck_checkports = true,
-  $ossec_rootcheck_checkfiles = true,
-  $ossec_server_ip            = undef,
-  $ossec_server_hostname      = undef,
-  $ossec_server_port          = '1514',
-  $ossec_server_protocol      = 'udp',
-  $ossec_scanpaths            = [],
-  $ossec_emailnotification    = 'yes',
-  $ossec_ignorepaths          = [],
-  $ossec_local_files          = $::wazuh::params::default_local_files,
-  $ossec_syscheck_frequency   = 43200,
-  $ossec_prefilter            = false,
-  $ossec_service_provider     = $::wazuh::params::ossec_service_provider,
-  $ossec_config_profiles      = [],
-  $selinux                    = false,
-  $agent_name                 = $::hostname,
-  $agent_ip_address           = $::ipaddress,
-  $manage_repo                = true,
-  $manage_epel_repo           = true,
-  $agent_package_name         = $::wazuh::params::agent_package,
-  $agent_package_version      = 'installed',
-  $agent_service_name         = $::wazuh::params::agent_service,
-  $manage_client_keys         = 'export',
-  $agent_auth_password        = undef,
-  $wazuh_manager_root_ca_pem  = undef,
-  $agent_seed                 = undef,
-  $max_clients                = 3000,
-  $ar_repeated_offenders      = '',
-  $enable_wodle_openscap      = true,
-  $wodle_openscap_content     = $::wazuh::params::wodle_openscap_content,
-  $service_has_status         = $::wazuh::params::service_has_status,
-  $ossec_conf_template        = 'wazuh/wazuh_agent.conf.erb',
+  $ossec_active_response       = true,
+  $ossec_rootcheck             = true,
+  $ossec_rootcheck_frequency   = 36000,
+  $ossec_rootcheck_checkports  = true,
+  $ossec_rootcheck_checkfiles  = true,
+  $ossec_server_ip             = undef,
+  $ossec_server_hostname       = undef,
+  $ossec_server_port           = '1514',
+  $ossec_server_protocol       = 'udp',
+  $ossec_server_notify_time    = '600',
+  $ossec_server_time_reconnect = '1800',
```

Comment:
Hi @rafaelfc-olx 

These proposed values are of great use for users, but it is better to leave these "undef" variables for each user to make their modification and use the default values that are in the installation.

JN

Reply:
sure, makes sense. BTW these are the current defaults (https://documentation.wazuh.com/2.0/user-manual/reference/ossec-conf/client.html#time-reconnect and https://documentation.wazuh.com/2.0/user-manual/reference/ossec-conf/client.html#notify-time), but its safer to leave undef by default for future releases.

Thanks